doc/cephfs/client-config-ref.rst

```diff
@@ -4,115 +4,115 @@
 
 ``client acl type``
 
-:Description: Set to ``"posix_acl"`` to enable POSIX ACLs.
+:Description: Set the ACL type. Currently, only possible value is ``"posix_acl"`` to enable POSIX ACL, or an empty string.
 :Type: String
-:Default: ``""`` (N/A)
+:Default: ``""`` (empty string)
 
 ``client cache mid``
 
-:Description: Set client cache mid-point.
+:Description: Set client cache midpoint. The midpoint splits the least recently used lists into a hot and warm list.
 :Type: Float
 :Default: ``0.75``
 
 ``client_cache_size``
 
-:Description: Set client cache object count maximum.
+:Description: Set the number of inodes that the client keeps in the metadata cache.
 :Type: Integer
 :Default: ``16384``
 
 ``client_caps_release_delay``
 
-:Description: Delay between capability releases.
+:Description: Set the delay between capability releases in seconds. The delay sets how many seconds a client waits to release capabilities that it no longer needs in case the capabilities are needed for another user space operation.
 :Type: Integer
 :Default: ``5`` (seconds)
 
 ``client_debug_force_sync_read``
 
-:Description: Always read synchronously (go to osds).
+:Description: If set to ``true``, clients read data directly from OSDs instead of using a local page cache.
 :Type: Boolean
 :Default: ``false``
 
 ``client_debug_getattr_caps``
 
-:Description: Check if MDS reply contains wanted caps.
+:Description: Check if the reply from the MDS contains required capabilities.
 :Type: Boolean
 :Default: ``false``
 
 ``client_dirsize_rbytes``
 
-:Description: Use recursive size of directory for directory ``st_size`` stat member.
+:Description: If set to `true`, use the recursive size of a directory (that is, total of all descendants).
 :Type: Boolean
 :Default: ``true``
 
 ``client_max_inline_size``
 
-:Description: Maximum size of inlined data stored in a file inode (rather than in a separate data object in RADOS). This setting only applies if the ``inline_data`` flag is set on the MDSMap.
+:Description: Set the maximum size of inlined data stored in a file inode rather than in a separate data object in RADOS. This setting only applies if the ``inline_data`` flag is set on the MDS map.
 :Type: Integer
 :Default: ``4096``
 
 ``client_metadata``
 
 :Description: Comma-delimited strings for client metadata sent to each MDS.
```

Comment:
Let's extend this description to say "...to each MDS, in addition to the automatically generated version, hostname, etc"